tests/infer/mcmc/test_hmc.py

```diff
@@ -123,7 +123,6 @@ def rmse(t1, t2):
     'fixture, num_samples, warmup_steps, hmc_params, expected_means, expected_precs, mean_tol, std_tol',
     TEST_CASES,
     ids=TEST_IDS)
-@pytest.mark.skip(reason='Slow test (https://github.com/pytorch/pytorch/issues/12190)')
```

Comment:
I think we should skip this on CI for sure. All of these together add several minutes to testing time and provide little in the way of additional information (the code paths they test are covered by faster tests). They are very useful for local profiling though, so if they all take more than a minute I would keep the skip flag.

Reply:
You are right! I should look at CI log before making this PR. So we'll keep this for profiling. I'll change the title for this PR though (clearer skip messages)